config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -271,6 +271,7 @@ periodics:
   annotations:
     testgrid-num-failures-to-alert: '2'
     testgrid-alert-stale-results-hours: '48'
-    testgrid-dashboards: conformance-all, conformance-kind, sig-testing-kind
+    testgrid-dashboards: conformance-all, conformance-kind, sig-testing-kind, sig-release-master-blocking
     testgrid-tab-name: conformance-ga-only
     testgrid-alert-email: liggitt@google.com
+    description: "conformance tests run against a master kind cluster with only GA APIs and features enabled"
```

Comment:
I _believe_ this description needs to start with: `OWNER: <owner> blah blah blah`
We should look at making these configurations more clear moving into 1.19.

Reply:
added